src/edu/csus/ecs/pc2/ui/ProblemsPane.java

```diff
@@ -281,6 +283,14 @@ protected Object[] buildProblemRow(Problem problem) {
         // c[i++] = new MCLBCenteredStringCellRenderer(Integer.toString(problem.getTimeOutInSeconds()));
         c[i++] = Integer.toString(problem.getTimeOutInSeconds());
 
+        // problem output limit
```

Comment:
This presents the user with invalid information.   If the max output limit is zero then the user sees another
value entirely.   If they then use the Problems report or export the yaml then zero will be output leaving
the user wondering what happened.   

Note: here is another discussion about the value of zero for max output size in K.

Reply:
I agree.  Instead of using "0" to mean "use the system global", perhaps have a check box that says: "Override maximum output for this problem", then an edit box next to that checkbox that becomes enabled when the checkbox is checked.  Basically, any place where you can enter 0 to "use the default" should maybe have a checkbox as described?  This is usually what we do in our GUI apps when the user has a choice to override something.  They have to make a conscious decision to override (checkbox), and then (and only then) can they type text in the override box. Just a suggestion.

Reply:
After meeting, an explanation was given that 0 for max in Problems column would be accurate but misleading.   New code added in Executable to change 0 to the model default max (from ContestInformation).

Also noted that in Edit Problem the max is displayed as is, so zero shows max output limit to be zero.   There also is a
description in the "What is this" icon next to the field (getLblWhatsThisMaxOutputSize())